packages/components/src/DataList/DataList.css

```diff
@@ -46,18 +50,54 @@
   margin: calc(var(--space-small) * -1) 0;
 }
 
+/*
+ * List Item
+ */
+
 .listItem {
   position: relative;
   padding: var(--space-small);
   border-bottom: var(--border-base) solid var(--color-border);
   transition: all var(--timing-base);
 }
 
-.listItem.active {
+.listItem.active,
```

Comment:
Should also use this visual state for when the `listItem` itself is focused - almost tripped me up when I was testing the tab order because it wasn't apparent my focus was set anywhere

Reply:
Update: it looks good now 👻